### ISSUE TITLE
Logos

### ISSUE THREAD

**Issue by vivkay** [vivkay/fw-silent-auction#4]
Hey @ccmadrid – do we want the sponsors' logos on the footer? I think it would be a nice touch. A nice to have?

**Comment by ccmadrid** [comment]
@vivkay only person to send a sponsor logo was for the subway card and nspire signs. I think it would be cool if we did to say thanks to the sponsors, but I don't know which cards are from sponsors or just donations from people you know? cuz there's a difference and I don't want to give credit to people who don't deserve it haha. so if we did, we'd need to make sure the sponsors are sponsors and get/make their assets.

**Comment by vivkay** [comment]
Good point. Perhaps we can include general text that says 'Thank you to all our Sponsors' in the footer (no images).
